src/test/java/uk/gov/pay/adminusers/utils/date/DisputeEvidenceDueByDateUtilTest.java

```diff
@@ -17,17 +20,16 @@ class DisputeEvidenceDueByDateUtilTest {
 
     @ParameterizedTest
     @MethodSource("epochProvider")
-    void shouldCorrectlyCalculatePayDueByDate(ZonedDateTime dateTime, DayOfWeek expected) {
-        assertThat(getPayDueByDate(dateTime).getDayOfWeek(), is(expected));
+    void shouldCorrectlyCalculatePayDueByDate(ZonedDateTime dateTime, DayOfWeek expectedDayOfWeek, Month expectedMonth, Year expectedYear) {
+        assertThat(getPayDueByDate(dateTime).getDayOfWeek(), is(expectedDayOfWeek));
+        assertThat(getPayDueByDate(dateTime).getMonth(), is(expectedMonth));
+        assertThat(getPayDueByDate(dateTime).getYear(), is(expectedYear.getValue()));
     }
 
     private static Stream<Arguments> epochProvider() {
         return Stream.of(
-                Arguments.of(parse("2022-03-07T13:00:00.000Z"), DayOfWeek.FRIDAY),
-                Arguments.of(parse("2022-03-08T13:00:00.000Z"), DayOfWeek.FRIDAY),
-                Arguments.of(parse("2022-03-09T13:00:00.000Z"), DayOfWeek.MONDAY),
-                Arguments.of(parse("2022-03-10T13:00:00.000Z"), DayOfWeek.TUESDAY),
-                Arguments.of(parse("2022-03-11T13:00:00.000Z"), DayOfWeek.WEDNESDAY)
+                Arguments.of(parse("2023-01-03T13:00:00.000Z"), DayOfWeek.TUESDAY, Month.DECEMBER, Year.of(2022))
+
```

Comment:
This does not need to be parameterised at all now, so you can just make it `@Test` rather than `@ParameterisedTest`. Furthermore, you can just create an expected `ZonedDateTime` and compare them using `assertThat(…)`. Like this:

```java
@Test
void shouldCorrectlyCalculatePayDueByDate() {
    ZonedDateTime dateTime = ZonedDateTime.parse("2023-01-03T13:00:00.000Z");
    ZonedDateTime expectedDueByDate = ZonedDateTime.parse("2022-12-27T13:00:00.000Z");

    ZonedDateTime actualDueByDate = getPayDueByDate(dateTime);

    assertThat(actualDueByDate, is(expectedDueByDate));
}
```